sql/pre_install/tables.sql

```diff
@@ -367,6 +367,45 @@ CREATE INDEX IF NOT EXISTS remote_txn_data_node_name_idx ON _timescaledb_catalog
 
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.remote_txn', '');
 
+-- This table stores information about the stage that has been completed of a
+-- chunk move/copy activity (cma)
+--
+-- Cleanup activity can query and check if the backend is running. If the
+-- backend has exited then we can commence cleanup. The cleanup
+-- activity can also do a diff with the "time_start" value to ascertain if
+-- the entire end-to-end activity is going on for too long
+--
+-- We also track the end time of every stage. A diff with the current time
+-- will give us an idea about how long the current stage has been running
+--
+-- Entry for a chunk move/copy activity gets deleted on successful completion
+--
+-- We don't want to pg_dump this table's contents
+--
+-- TODO: check the locking requirements
+--
+-- TODO: more brainstorming for the logic to check that CMA is ongoing/aborted/completed
+
+CREATE SEQUENCE IF NOT EXISTS _timescaledb_catalog.chunk_move_activity_id_seq;
+
+CREATE TABLE _timescaledb_catalog.chunk_move_activity (
+  id integer PRIMARY KEY DEFAULT nextval('_timescaledb_catalog.chunk_move_activity_id_seq'),
+  operation_id name NOT NULL UNIQUE, -- the publisher/subscriber identifier used
+  backend_pid integer NOT NULL, -- the pid of the backend running this cma
+  stage text NOT NULL, -- the complete stage/step of the CMA
+  time_start timestamptz NOT NULL DEFAULT NOW(), -- start time of the cma
+  time_end timestamptz, -- end time of the cma. Means completed
+  time_prev_stage_end timestamptz NOT NULL, -- end time of this stage,
+                                            -- implies start time of next stage, if any
+  chunk_id integer NOT NULL REFERENCES _timescaledb_catalog.chunk (id) ON DELETE CASCADE,
+  source_node_name name NOT NULL,
+  dest_node_name name NOT NULL,
+  is_move bool NOT NULL -- is a move or copy activity
+);
+
+ALTER SEQUENCE _timescaledb_catalog.chunk_move_activity_id_seq OWNED BY _timescaledb_catalog.chunk_move_activity.id;
+SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.chunk_move_activity_id_seq', '');
```

Comment:
I think you're missing a `pg_catalog.pg_extension_config_dump` for `_timescaledb_catalog.chunk_move_activity`:
```suggestion
SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.chunk_move_activity_id_seq', '');
SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.chunk_move_activity', '');
```